lib/executors/Executor.js

```diff
@@ -297,10 +297,6 @@ define([
 				})
 				.finally(lang.bind(self, '_afterRun'))
 				.then(function () {
-					if (self._hasSuiteErrors) {
-						throw new Error('One or more suite errors occurred during testing');
```

Comment:
Should fatal errors be thrown if a suite error has occured? I would assume not, as suite errors have their own handlers. Second opinions?